src/playground/services/invoices/update/view.clj

```diff
@@ -12,12 +12,15 @@
     [:h1 "All Entries"]
     [:div
      (let [attr-fns {:data-value-transform
-                     (fn [label-key v]
-                       (if (= :id label-key)
-                         [:a {:href (str "/invoices/" v)} v]
-                         v))}]
-       (table/to-table1d
-        (:result context)
-        [:id "ID" :email "Email"]
-        attr-fns))]]))
+                       (fn [label-key v]
+                         (if (= :id label-key)
+                           [:a {:href (str "/invoices/" v)} v]
```

Comment:
Avoid hardcoding these strings

Instead use `[io.pedestal.http.route :refer [url-for]]`

Usage is simply `(url-for :invoices-delete)`, where `:invoices-delete` is the `:route-name` defined like this:

```
["/echo" :get #(hash-map :body % :status 200) :route-name :echo]
```

Reply:
I don't quite follow.. Looked at the example in pedestal site but I am not sure how this is implemented.

Reply:
You can use my example code quite literally

Reply:
copying this route (as is) and adding the`url-for` dependency to the view, then replacing the `str` statement with any variation of `url-for :,,,` gives me a "Route not found" exception on render.
How do you test `url-for` in the repl? whatever I try returns: 
` url-for not bound `

Reply:
For clarity, you're not supposed to copy `["/echo" :get #(hash-map :body % :status 200) :route-name :echo]`. The point is that you can add `:route-name :xxxxxxxxx`

If it still doesn't work, commit your attempt and let me see